ScriptLinkStandard.Tests/HelpersTests/TransformToXmlTests.cs

```diff
@@ -125,7 +125,7 @@ public void RowObjectTransformToXmlIsString()
         public void RowObjectHelperTransformToXmlIsString()
         {
             string expected = "<?xml version=\"1.0\" encoding=\"utf-16\"?>" + Environment.NewLine
-                            + "<RowObject xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">" + Environment.NewLine
```

Comment:
This passes for me before the change and matches the [Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/standard/serialization/xml-serialization-with-xml-web-services). Have you modified the XML Serialization or the .NET version?

Reply:
I don't believe I made any modifications to either.  That's strange.